app/views/layouts/application.html.erb

```diff
@@ -23,6 +23,10 @@
   <body>
     <%= link_to 'Sign out', destroy_user_session_path, :method=>'delete' %>
 
+    <% if current_user %>
+      <%= link_to 'My albums', ('/users/' + current_user.id.to_s + '/albums') %>
```

Comment:
you should be able to user a rails path helper here `users_albums_path(user)` something like this